configs/ttalps_histogrammer_list.py

```diff
@@ -107,7 +117,7 @@
 sample_path = ""
 input_directory = f"{base_path}/{sample_path}/{skim}/"
 
-output_dir = f"{input_directory}/histograms"
+output_dir = f"{input_directory}/histograms".replace("lrygaard", "jniedzie")
```

Comment:
Do you think we can make this automatic? Or make it clear somewhere so I don't accidentally start to try to write to your space?

Reply:
yeah, we should probably read the user name automatically in this script and paste it there. For input data, we will still have to do it by hand though